README.md

```diff
@@ -1,137 +1,143 @@
 
-## Actions
-### Split Message By Array
-**This action is deprecated, please use Split on JSONata Expression instead.**
-
-Splits a message into multiple messages using a given separator. The separator is treated as a path to a property inside the message. A message is split when a property is an array and emitted are multiple messages. Otherwise the original message is emitted.
-
-For example, we have a message:
-
-```
-{
-    "users": [
-        {
-            "name": "John"
-        },
-        {
-            "name": "Mike"
-        }
-    ]
-}
-```
-
-The splitting expression is "users", action will return output:
-```
-{
-    "name": "John"
-}
-
-{
-    "name": "Mike"
-}
-```
-*Notes:*
-
-- *When splitting expression refers to an object splitter returns this object;*
-- *When splitting expression contains primitive value like ```users:"John"``` or array of primitives like ```users:["John", "Mike", "Anna"]``` splitter emits error.*
+The Splitter Component is a fundamental part of our platform. Its primary purpose is to split arrays into separate messages or to combine separate messages into one.
 
+## Actions 
+  
 ### Split on JSONata Expression
 
-This component takes the incoming message body and applies the configured JSONata transformation on it. The evaluated transformation must be an array. This array is split and emitted into multiple messages.
-
-For example, given the following message:
-
-```
-{
-    "FirstName": "Fred",
-    "Surname": "Smith",
-    "Phone": [
-        {
-            "type": "home",
-            "number": "0203 544 1234"
+Splits a single message into several separate messages.
+
+#### Configuration Fields
+
+* **JSONata Expression** - (string, required): Enter the expression that will be evaluated as an array.
+
+ <details><summary>Example</summary>
+
+ You have the following incoming message:
+
+  ```json
+    {
+        "FirstName": "Fred",
+        "Surname": "Smith",
+        "Phone": [
+            {
+                "type": "home",
+                "number": "0203 544 1234"
+            },
+            {
+                "type": "office",
+                "number": "01962 001234"
+            },
+            {
+                "type": "mobile",
+                "number": "077 7700 1234"
+            }
+        ]
+    }
+  ```
+
+  If the JSONata expression is set to `Phone.{type: number}`, you will get three messages:
+  
+  ```json
+    {
+        "home": "0203 544 1234"
+    }
+  ```
+  ```json
+    {
+        "office": "01962 001234"
+    }
+  ```
+  ```json
+    {
+        "mobile": "077 7700 1234"
+    }
+  ```
+  </details> <br>
+
+#### Input Metadata
+
+None
+
+#### Output Metadata
+
+Each item of the array will be emitted as a separate message.
+
+### Re-assemble Messages
+
+Combines separate messages into one.
+
+#### Configuration Fields
+
+* **Behavior** - (dropdown, required): Select one of the following options:
```

Comment:
It makes sense to mention the time to wait before emit here in the options below

Reply:
done